### PR TITLE
Server feature updates

### PR DESCRIPTION
**Pull request by cmertenscm**
MySQL data source testing, more unit testing.

### CODE REVIEW

File: client-src/package.json.orig [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "vinyl-buffer": "^1.0.0",
+    "vinyl-source-stream": "^1.1.0",
+    "watchify": "^3.7.0",
+    "yargs": "^3.29.0"
+  },
+  "dependencies": {
+<<<<<<< HEAD
+    "@asaayers/react-a11y": "^0.1.0",
+=======
+>>>>>>> origin/develop
```
Comment (mmccloskeyacn) [inline client-src/package.json.orig]: Merge junk?

**Comment by cmertenscm** [comment]
Nice catch; I thought I had cleared all of that out. I'll clean and commit.

File: client-src/package.json.orig (hunk 1 above)
Comment (cmertenscm) [inline client-src/package.json.orig]: Good catch; removed from repo.

**Comment by mmccloskeyacn** [comment]
Approved!

**Pull request merged**